services-camel/pom.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<artifactId>cdr</artifactId>
+		<groupId>edu.unc.lib.cdr</groupId>
+		<version>4.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>services-camel</artifactId>
+	<version>1.0</version>
+	<packaging>war</packaging>
+
+	<name>thumbnail-enhancements</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<fcrepo-camel.version>4.5.0</fcrepo-camel.version>
+		<camel.version>2.17.3</camel.version>
+		<activemq.version>5.14.1</activemq.version>
+		<osgi.export.packages>edu.unc.lib.cdr.thumbnail;version=${project.version}</osgi.export.packages>
+		<osgi.import.packages>*</osgi.import.packages>
+	</properties>
+
+	<dependencies>
+
+	<!-- Camel -->
+	<dependency>
+		<groupId>org.apache.camel</groupId>
+		<artifactId>camel-core</artifactId>
+		<version>${camel.version}</version>
+	</dependency>
+	<dependency>
+		<groupId>org.apache.camel</groupId>
+		<artifactId>camel-spring</artifactId>
+		<version>${camel.version}</version>
+	</dependency>
+	<dependency>
+		<groupId>org.apache.camel</groupId>
+		<artifactId>camel-exec</artifactId>
+		<version>${camel.version}</version>
+	</dependency>
+	<dependency>
+	<groupId>org.apache.camel</groupId>
+		<artifactId>camel-http</artifactId>
+		<version>${camel.version}</version>
+	</dependency>
+	<dependency>
+		<groupId>org.apache.camel</groupId>
+		<artifactId>camel-spring-javaconfig</artifactId>
+		<version>${camel.version}</version>
+	</dependency>
+	<dependency>
+		<groupId>org.apache.activemq</groupId>
+		<artifactId>activemq-camel</artifactId>
+		<version>${activemq.version}</version>
+	</dependency>
+
+	<dependency>
+		<groupId>org.fcrepo.camel</groupId>
+		<artifactId>fcrepo-camel</artifactId>
+		<version>${fcrepo-camel.version}</version>
+		<exclusions>
```

Comment:
Looks like we don't need this exclusion anymore since we bumped up the veresion of fcrepo-camel-toolbox